docs/dev/testing.md

```diff
@@ -0,0 +1,59 @@
+<!--Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan 
+Author(s): Marina Schmidt 
+Modifications:
+
+Disclaimer:
+pyDARN is under the LGPL v3 license found in the root directory LICENSE.md 
+Everyone is permitted to copy and distribute verbatim copies of this license 
+document, but changing it is not allowed.
+
+This version of the GNU Lesser General Public License incorporates the terms
+and conditions of version 3 of the GNU General Public License, supplemented by
+the additional permissions listed below.
+-->
+
+
+# Testing Pull Requests 
+
+One way to help pyDARN get developed faster and smoother is by testing [Pull Requests](https://github.com/SuperDARN/pydarn/pulls). 
+This can take fifteen minutes or less to do, helps the developer merge their code,
+and allows you to be on the author list!
+
+Every contribution helps! 
+
+## Setting up the testing environment
+
+If you haven't cloned pyDARN before this will get you the correct branch installed to just test:
+`pip install git+https://github.com/superdarn/pydarn@<branchname>`
+Make sure to install any new instances of pyDARN, or branches that are not master in a virtual environment to avoid mixing pyDARN versions! 
+
+
+If you wish to clone pyDARN for multi-branch comparison or testing in the future: 
```

Comment:
```suggestion
If you wish to clone pyDARN for multi-branch comparison or testing in the future: 

```

Make the list in mkdocs otherwise it's all in a paragraph